helpers/send_report_to_admins.js

```diff
@@ -0,0 +1,64 @@
+module.exports = function(type) {
+  var nodemailer = require('nodemailer'),
+  smtpTransport = require('nodemailer-smtp-transport'),
+  markdown = require('nodemailer-markdown').markdown,
+  Admin = require('../models/admin'),
+  BugReport = require('../models/bug_report'),
+  CrashReport = require('../models/crash_report');
+
+  return new Promise((function(resolve, reject) {
+    Admin.find(function(error, admins) {
```

Comment:
Before this, it should check when the last report was sent (has to be one for bug and one for crash reports). If it's no more than X hours ago, then just `reject` it.